src/officehours_api/backends/zoom.py

```diff
@@ -118,6 +118,13 @@ def _get_access_token(cls, user: User) -> str:
                 },
                 headers=cls._get_client_auth_headers(),
             )
+            if resp.status_code == 401:
+                # The refresh_token was invalidated somehow.
+                # Maybe the user removed our Zoom app's auth.
+                # Force them to be prompted again.
+                zoom_meta = {}
+                user.profile.backend_metadata['zoom'] = zoom_meta
+                user.profile.save()
```

Comment:
Yeah, so this seems to work when I remove the app integration manually. It is triggered when you hit "Create Meeting". There is still an error logged in the backend; that may not be desirable, but I don't expect this to happen very often.